src/nycdb/sql.py

```diff
@@ -6,7 +6,33 @@ def create_table(table_name, fields):
 
 
 def insert(table_name, row):
+    '''
+    Generate a psycopg2-friendly string that uses placeholders to
+    insert the given row into the database.  This can be passed to
+    cursor.execute() as its first parameter; you'll want to pass the
+    row itself as the second parameter.
+    '''
```

Comment:
It took me a while to realize that this function wasn't returning a literal SQL string (I was like "what about bobby tables?!?!!") so I figured I'd document that here.